### PR TITLE
21616 Add alt text to mdn-canvas-lineargradient.png

### PR DESCRIPTION
**Pull request by pelnik**
### Description

Related to issue #21616 Added alt text for linear gradient image

### Motivation

Needed for open issue

### CODE REVIEW

File: files/en-us/web/api/canvasrenderingcontext2d/createlineargradient/index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The
 method of the Canvas 2D API creates a gradient along the line connecting two given
 coordinates.
 
-![](mdn-canvas-lineargradient.png)
+![An example of the application of the gradient applied to a shape](mdn-canvas-lineargradient.png)
```
Comment (estelle) [inline files/en-us/web/api/canvasrenderingcontext2d/createlineargradient/index.md:22]: How about:

```suggestion
![The gradient goes transitions colors along the gradient line, starting at point x0, y0 and going to x1, y1, even if those points extend the gradient line beyond the edges of the element on which the gradient is drawn.](mdn-canvas-lineargradient.png)
```

The user who can't see the image needs to be able to understand all the information the image conveys; not what the image is. This also helps users who can see the image but don't understand what it is conveying.

**Review by estelle — COMMENTED** [review]
made a suggestion

File: files/en-us/web/api/canvasrenderingcontext2d/createlineargradient/index.md (hunk 1 above)
Comment (OGBrisket) [reply files/en-us/web/api/canvasrenderingcontext2d/createlineargradient/index.md]: Thank you @estelle! I removed the extra work "goes" and commited this. This explanation will help with my future commits on this issue.

**Review by Elchi3 — APPROVED** [review]
Thank you both! 👍

**Pull request merged**